plugins/decklink/decklink-device-instance.cpp

```diff
@@ -639,20 +632,30 @@ HRESULT STDMETHODCALLTYPE DeckLinkDeviceInstance::VideoInputFormatChanged(
 		}
 	}
 
-	const HRESULT videoResult = input->EnableVideoInput(
-		displayMode, pixelFormat, bmdVideoInputEnableFormatDetection);
-	if (videoResult != S_OK) {
-		LOG(LOG_ERROR, "Failed to enable video input");
-		input->StopStreams();
-		FinalizeStream();
+	if (events & bmdVideoInputDisplayModeChanged) {
 
-		return E_FAIL;
-	}
+		input->PauseStreams();
+
+		mode->SetMode(newMode);
+
+		displayMode = mode->GetDisplayMode();
```

Comment:
I don't see a particular need to separate these individual lines with blank lines.

Reply:
removed